oarepo_ui/resources/components.py

```diff
@@ -270,3 +272,34 @@ def before_ui_edit(self, *, api_record, extra_context, identity, **kwargs):
 
     def before_ui_detail(self, **kwargs):
         self.before_ui_edit(**kwargs)
+
+
+class AllowedHtmlTagsComponent(UIResourceComponent):
+    def calculate_valid_tags_for_editor(self, tags, attr):
```

Comment:
I would leave this utility function purely in JS (format it outputs is relevant only to a concrete JS widget and can be easily derived from other 2 config properties, and changed, whenever editor widget changes)